python/tvm/tir/transform/transform.py

```diff
@@ -402,7 +402,9 @@ def MakePackedAPI(num_unpacked_params: int = -1):
     fpass : tvm.transform.Pass
         The result pass
     """
-    return _ffi_api.MakePackedAPI(num_unpacked_params)  # type: ignore
+    num_unpacked_params: int = -1
+    assert num_unpacked_params == -1
```

Comment:
I think we can remove this now?

Reply:
Thank you for the catch, and removed!  I was iteratively stripping out `num_unpacked_params` to make sure that each step still ran, and it looks like that one slipped by.